packages/css/src/components/Toggle.css

```diff
@@ -0,0 +1,113 @@
+.Toggle {
+  position: relative;
+  width: 3.375rem /* 54px */;
+  height: 1.25rem /* 20px */;
+  background-color: var(--color-white);
+  border: 1px solid var(--color-chatelle);
+  border-radius: 10px;
+  cursor: pointer;
+  transition: all 0.3s;
+}
+
+.Toggle:hover {
+  border: 1px solid var(--color-granite);
+}
+
+.Toggle-bullet {
+  position: absolute;
+  height: 0.875rem /* 14px */;
+  width: 0.875rem /* 14px */;
+  background-color: var(--color-granite);
+  border-radius: 8px;
+  top: 2px;
+  left: 2px;
+  transition: all 0.3s;
+}
+
+.Toggle-bullet-label {
+  text-transform: uppercase;
+  text-align: center;
+  font-size: var(--font-size--xs);
+  line-height: var(--line-height--s);
+  position: absolute;
+  top: 2px;
+  bottom: 2px;
+}
+
+.Toggle-bullet-label.Toggle-bullet-label--off {
+  left: 18px;
+  right: 4px;
+  color: var(--color-granite);
+  visibility: visible;
+  transition: visibility 0.3s linear;
+  transition-delay: 0.1s;
+}
+
+.ToggleContainer.is-on .Toggle-bullet-label--off {
+  visibility: hidden;
+  transition: visibility 0.1s linear;
+}
+
+.Toggle-bullet-label--on {
+  right: 18px;
+  left: 4px;
+  color: var(--color-white);
+  visibility: hidden;
+  transition: visibility 0.1s linear;
+}
+
+.ToggleContainer.is-disabled .Toggle-bullet-label--on {
+  color: var(--color-granite);
+}
+
+.ToggleContainer.is-on .Toggle-bullet-label--on {
+  visibility: visible;
+  transition: visibility 0.3s linear;
+  transition-delay: 0.1s;
+}
+
+.ToggleContainer.is-on .Toggle {
+  background-color: var(--color-rose);
+  border: 1px solid var(--color-rose);
+  transition: all 0.3s;
+}
+
+.ToggleContainer.is-on .Toggle-bullet {
+  left: 100%;
+  margin-left: -16px;
+  transition: all 0.3s;
+  background-color: var(--color-white);
+}
+
+.ToggleContainer.is-disabled .Toggle {
+  background-color: var(--color-grey);
+  border: 1px solid var(--color-grey);
+  cursor: not-allowed;
+}
+
+.ToggleContainer.is-disabled .Toggle-bullet {
+  background-color: var(--color-chatelle);
+}
+
+.ToggleContainer {
+  display: flex;
+}
+
+.ToggleContainer > .Toggle {
+  margin-right: var(--space-l);
```

Comment:
This should go with `.Toggle-labels`, so you don't have to worry about `Toggle` margin.
Adding labels will add margins. If there is no label, there is no margin 
And the specificity will be highly reduced ✨

Reply:
I did it this way b/c even if there's no label I wanted to keep some horizontal distance between toggles if they were to be next to another. Even though it's doing it's job, I'm aware it's not very `all in place and order`. How about if I more this to the left of `Toggle-labels` and add small margin on left and right of `ToggleContainer`?

Reply:
👍